pkg/defkinds/nodejs/definition.go

```diff
@@ -262,6 +285,17 @@ func (s Stage) Merge(overriding Stage) Stage {
 	return new
 }
 
+var defaultHealthcheck = builddef.HealthcheckConfig{
+	HealthcheckHTTP: &builddef.HealthcheckHTTP{
+		Path:     "/ping",
```

Comment:
this is used for an express server ?
do we need HC on nodejs kinds as they'll use a webserver up front (nginx `/_status`) ?

Reply:
Yep this is only for nodejs backends/workers use cases. I need to replace the `frontend: true` param by a new `mode: frontend|server|worker`. Once done I'll improve the way the default healthcheck is defined :wink: